translations/ja/lab5.md

```diff
@@ -67,10 +65,10 @@ functions:
       write_debug: true
 ```
 
-* functionをデプロイします
+* functionをビルド、プッシュ、デプロイします
 
 ```
-$ faas-cli deploy -f ./issue-bot.yml
```

Comment:
This one probably shouldn't change

Reply:
The line is deleted in this commit.
https://github.com/openfaas/workshop/commit/f27ff19c87b0933c51799586d83d5e75cbd78eb2#diff-9852b83ba9abfceda99773422ef371ac

And I think it will be deployed L72.
```
$ faas-cli up -f ./issue-bot.yml
```

Reply:
It looks like the original version has changed, too.
https://github.com/openfaas/workshop/commit/f27ff19c87b0933c51799586d83d5e75cbd78eb2#diff-9852b83ba9abfceda99773422ef371acR72

But yes, this only needs `deploy` because the environment variable is the only thing that changed. Perhaps we should revert it back to `faas-cli deploy -f ./issue-bot.yml`.